addons/website_sale/models/website_snippet_filter.py

```diff
@@ -186,14 +189,13 @@ def _get_products_recently_sold_with(
                         domain,
                         [('id', 'in', products_ids)],
                     ])
-                    products = self.env['product.product'].with_context(
+                    products = products.with_context(
```

Comment:
Useless diff

Reply:
` self.env['product.product']` is already defined with `products` so it is good to reuse it.

Reply:
Hello @Feyensv 
Should I remove this diff?

Reply:
IMO, It doesn't make the code clearer, loses part of the blame, and is really negligible in the little gains it can bring, so yes :D